README.md

```diff
@@ -19,10 +19,10 @@ Solidity is a statically typed, contract-oriented, high-level language for imple
 Solidity is a statically-typed curly-braces programming language designed for developing smart contracts
 that run on the Ethereum Virtual Machine. Smart contracts are programs that are executed inside a peer-to-peer
 network where nobody has special authority over the execution, and thus they allow to implement tokens of value,
-ownership, voting and other kinds of logics.
+ownership, voting, and other kinds of logic.
```

Comment:
Are you sure the singular is correct here?

Reply:
I think the plural or logic is logic. Logics also seems to exist according to https://www.wordhippo.com/what-is/the-plural-of/logic.html.

Reply:
As the word "kind" is used in its plural form "kinds", "logic" need not be used in its plural form.